### DISCUSSION TITLE
What does FIPS Status: unverified mean?

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [goauthentik/authentik#11824] [Q&A] [answered]
Heho
My authentik dashboard shows "FIPS Status: unverified".
What is this and how do I get this to be green?
![Bildschirmfoto vom 2024-10-26 15-59-13](https://github.com/user-attachments/assets/deca957c-954f-4832-b6d0-e67e7faf7608)

**Comment by jrndberg** [comment]
+1

**Comment by fheisler** [comment, marked as answer]
[FIPS](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards) is a set of standards used mostly in the context of the US federal government and its vendors/customers. Right now, licensed instances see this API check by default, which verifies that only FIPS-validated modules are installed/used across the instance. This mode needs to be manually enabled at build time, and we work with a few customers who require this feature to enable it.

From a practical standpoint, if you don't care about this specific compliance standard, whether the check passes does not negatively affect the security of your installation. (We can actually use _more_ up-to-date packages when not limited to FIPS-validated modules.)

Longer term, we will make the appearance of the admin dashboard more configurable. At that point, we would also make it optional to show or hide this FIPS validation check.
